traffic-policy/actions/terminate-tls/variables.mdx

```diff
@@ -0,0 +1,69 @@
+| `actions.ngrok.terminate_tls.server.subject.common_name`         | `string`      | Common name of the subject, usually the domain name.                                                                |
+| `actions.ngrok.terminate_tls.server.subject.country`             | `[]string`    | Country name(s) where the subject of the certificate is located.                                                    |
+| `actions.ngrok.terminate_tls.server.subject.locality`            | `[]string`    | Locality or city where the subject is located.                                                                      |
+| `actions.ngrok.terminate_tls.server.subject.organization`        | `[]string`    | Name(s) of the organization to which the subject belongs.                                                           |
+| `actions.ngrok.terminate_tls.server.subject.organizational_unit` | `[]string`    | Division of the organization to which the subject belongs.                                                          |
+| `actions.ngrok.terminate_tls.server.subject.postal_code`         | `[]string`    | Postal code where the subject is located.                                                                           |
+| `actions.ngrok.terminate_tls.server.subject.province`            | `[]string`    | Province or state where the subject is located.                                                                     |
+| `actions.ngrok.terminate_tls.server.subject.street_address`      | `[]string`    | Street address where the subject is located.                                                                        |
+| `actions.ngrok.terminate_tls.server.subject`                     | `string`      | The entity to whom the certificate is issued as a string roughly following the RFC 2253 Distinguished Names syntax. |
+| `actions.ngrok.terminate_tls.server.validity.not_after`          | `timestamp`   | Expiration date and time when the certificate is no longer valid.                                                   |
+| `actions.ngrok.terminate_tls.server.validity.not_before`         | `timestamp`   | Start date and time when the certificate becomes valid.                                                             |
+| `actions.ngrok.terminate_tls.sni`                                | `string`      | The hostname included in the `ClientHello` message via the SNI extension.                                           |
+| `actions.ngrok.terminate_tls.version`                            | `string`      | The version of the TLS protocol used between the client and the ngrok edge.                                         |
+
+**Note:** These variables are also available on the `conn` object (see [HTTP Variables](/docs/http/traffic-policy/expressions/variables/) for examples).
```

Comment:
Ope, there's also `actions.ngrok.terminate_tls.warnings`, `actions.ngrok.terminate_tls.error_code`, and `action.ngrok.terminate_tls.error_message`